cu-calendar.sty

```diff
@@ -112,13 +112,13 @@
 }%
 %
 \def\cuDayName#1{%
-  \ifcase#1 воскресение\or%
-    понедельник\or
-    вторник\or
-    среда\or
-    четверг\or
-    пятница\or
-    суббота\fi
+  \ifcase#1 недѣ́лѧ\or%
+    понедѣ́льникъ\or
+    вто́рникъ\or
+    среда̀\or
+    четверто́къ\or
+    пѧто́къ\or
+    сꙋббѡ́та\fi
 }%
```

Comment:
Nice. How about this solution then: we will provide \cuDayName (nominative) and \cu\DayNameAccusative (accusative) as two separate commands. User can define date using one or another, depending on her needs.

Alternative would be to make \cuDate and \cuJulianDate variants (\cuDateAccusative and \cuJulianDateAccusative)
that would automatically substitute \cuDayName with the \cuDayNameAccusative.

The latter seems cleaner.

Any ideas, comments?